modoboa_radicale/tests.py

```diff
@@ -3,10 +3,15 @@
 """
 import os
 import tempfile
-from ConfigParser import SafeConfigParser
+
+try:
+    from configparser import ConfigParser as SafeConfigParser
+except ImportError:
+    from ConfigParser import SafeConfigParser
```

Comment:
I thought six had a replacement for that?

Reply:
`six` can handle the module rename (`six.move.configparser`) however it doesn't handle the fact that `SafeConfigParser` (Py2) was renamed to `ConfigParser` (Py3).

If `six` is used something like the next code block is required, I prefer to handle it at the import stage and if I have to use a try/except block there's not much point in throwing six into the mix, although it is required later to handle the method renames (`read_fp` ->`read_file`).
```python
from six.move import configparser
...
cfg = configParser.ConfigParser() if six.PY3 else: configparser.SafeConfigParser()
```

To make future upgrades to a python 2 free life, I've changed as below and renamed all the calls to `SafeConfigParser` to `ConfigParser`
```python
try:
    from configparser import ConfigParser
except ImportError:
    # SafeConfigParser (py2) == ConfigParser (py3)
    from ConfigParser import SafeConfigParser as ConfigParser
```